torch/csrc/distributed/autograd/context/context.cpp

```diff
@@ -193,6 +202,25 @@ std::shared_ptr<rpc::FutureMessage> DistAutogradContext::
   return state->future;
 }
 
+void DistAutogradContext::setGraphTaskException(const std::string& errorMsg) {
```

Comment:
nit: can we use this function in `addOutstandingRpc` in the futureMessage callback?